src/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -5,7 +5,7 @@
     <ProjectGuid>{3CA89D6C-F8D1-4813-9775-F8D249686E31}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
-    <Configurations>netcoreapp-Linux-Debug;netcoreapp-Linux-Release;netcoreapp-OSX-Debug;netcoreapp-OSX-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
+    <Configurations>netcoreapp-Linux-Debug;netcoreapp-Linux-Release;netcoreapp-OSX-Debug;netcoreapp-OSX-Release;netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
```

Comment:
Do you know if these have to be static and directly in the project? I'm asking mainly to try and determine if we can combine the Debug/Release aspects with our existing BuildConfigurations property to eliminate the duplication.

Reply:
No I don’t know for sure. @davkean or @Pilchie - do you know off the top of your head?

Reply:
I believe it has to be known at evaluation time (i.e. Not the result of a target), but doesn't have to be directly in the project.  @davkean would know for sure.